README.md

```diff
@@ -153,7 +154,21 @@ Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed inf
   <p>Well, logically, you can't access anything else other than your Enterprise, but FastHub made that possible but can't do much about it,
   in most cases since your login credential doesn't exists in GitHub server. But in <b>few</b>
   cases your GitHub account Oauth token will do the trick.</p>
+</details>
+
+<details>
```

Comment:
editing an issue is a public...

Reply:
so sorry, but i don't understand what you are talking about....

Reply:
Test

Reply:
@yakov116 reply, please? I don't understand what is going on  here